miqcli/utils/__init__.py

```diff
@@ -124,3 +125,26 @@ def get_client_api_pointer():
         return click.get_current_context().find_root().client_api
     except AttributeError:
         log.error('Unable to get client api pointer.')
+
+
+def is_default_config_used():
+    """Is the default configuration used?
+
+    :return: True - defaults used, False - defaults not used
+    :rtype: bool
+    """
+    status = True
+
+    # get parent context
+    ctx = click.get_current_context().find_root()
+
+    # compare default config settings with final parameters
+    for key, value in ctx.default_map.items():
```

Comment:
I think this won't hurt - the only thing is, if my config is similar to the default config used in CLI I will receive this warning :-)